### PR TITLE
subway map css fixups

### PR DESCRIPTION
**Pull request by lbai001**
changed css for /disability-benefits/claims-appeal/

### CODE REVIEW

File: src/sass/base/_va.scss [hunk 1]
```diff
@@ -1397,37 +1397,41 @@ p.info-message {
...
   border: none;
-  border-left: 3px solid $color-gray-light;
+  border-left: 4px solid #cccccc;
   box-sizing: border-box;
   border-radius: 0;
   padding-left: 20px;
   padding-right: 0px;
   padding-bottom: 20px;
   padding-top: 0px;
+  margin-bottom: -30px;
```
Comment (cvalarida) [inline src/sass/base/_va.scss]: Looks like `.list-group-item` is used in `claims-appeals.md`, the discharge wizard's guidance page, and the discharge wizard's "request dd214" page. I tested the guidance page and claims-appeals:

### Guidance Page
image

### Appeals
image

I'm not sure how to get to the dd214 page without digging into it too much, but I'm going to guess it's probably okay there too.

**Review by cvalarida — COMMENTED** [review]
Looks great! Thanks for cleaning up the place while you were at it. I appreciate the attention to detail.

My only concern is moving away from variables for colors. I'd prefer using pre-defined color variables, but if we need these colors specifically, can you create new variables for them?

File: src/sass/base/_va.scss [hunk 2]
```diff
@@ -1397,37 +1397,41 @@ p.info-message {
 
 .vertical-list-group .list-group-item {
   border: none;
-  border-left: 3px solid $color-gray-light;
+  border-left: 4px solid #cccccc;
```
[the remark this answers is not in the document]
Comment (lbai001) [reply src/sass/base/_va.scss]: check

File: src/sass/base/_va.scss (hunk 1 above)
Comment (lbai001) [reply src/sass/base/_va.scss]: please review /discharge-upgrade-instructions/request-dd214 for request-dd214

File: src/sass/base/_va.scss [hunk 3]
```diff
@@ -1397,37 +1397,41 @@ p.info-message {
...
 .vertical-list-group .list-group-item:last-child {
-  border-left: 3px solid transparent;
+  border-left: 4px solid transparent;
   padding-bottom: 0;
 }
 .vertical-list-group .list-group-item::before {
   border-radius: 50%;
-  background-color: $color-gray-medium;
+  border: 3px #ffffff solid;
+  background-color: #757575;
```
Comment (cvalarida) [inline src/sass/base/_va.scss]: A couple more...

File: src/sass/modules/_m-modal.scss [hunk 4]
```diff
@@ -29,7 +29,7 @@
     }
 
     &-undefined {
-      background: white;
+      background: #ffffff;
```
Comment (cvalarida) [inline src/sass/modules/_m-modal.scss]: Last one.

**Review by cvalarida — COMMENTED** [review]
Looks like I missed a few last time, sorry about that. :grimacing: 

You can find a pretty comprehensive list of color variables in [`src/sass/base/_variables.scss`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/sass/base/_b-variables.scss). Feel free to add your own variables to the top of the individual app's scss file, but we try to use predefined ones where we can to keep our style consistent site-wide (no easy feat!).

**Review by cvalarida — APPROVED** [review]
Looks great! :rocket:

**Pull request merged**